Valigator.AspNetCore3.IntegrationTests/IntegrationTests.cs

```diff
@@ -99,5 +99,83 @@ public Task ObjectWithNonDataPropertyTest()
 				{
 					str.Should().Be("{\"IdentifierCollection\":[{\"TheIdentifier\":\"61f45cfd-6389-4380-a803-c23881e982af\"}],\"SecondValue\":1}");
 				});
+
+		[Fact]
+		public Task MappedGuidEndpointReturnsProperError()
+			=> CreateClient()
+				.BuildTest()
+				.Post("mappedGuid/mappedGuid")
+				.WithJsonBody($"{{\"Items\":[{{\"MappedGuid\":\"NotAGuid\"}}]}}")
+				.Send()
+				.IsBadRequest()
+				.AssertJsonBody(str =>
+				{
+					str
+						.Should()
+						.Match<string>(str => 
```

Comment:
The Match is used so that it can be either the Newtonsoft or System.Text formatted version of the message.